src/App.js

```diff
@@ -1,10 +1,18 @@
 import React from 'react';
 import './App.css';
 
+const onClickReaction = massage => alert('Button was clicked!');
```

Comment:
`handleClick`
`showMessage`
`showAlert`
function name has to describe action, not the result